src/RequestHasher.php

```diff
@@ -27,7 +27,7 @@ public function __construct(CacheProfile $cacheProfile)
     public function getHashFor(Request $request)
     {
         return 'laravel-responsecache-'.md5(
-            "{$request->getUri()}/{$request->getMethod()}/".$this->cacheProfile->cacheNameSuffix($request)
+            "{$request->getUri()}/".$this->cacheProfile->cacheNameSuffix($request)
```

Comment:
please add surround concatenation (`.`) with spaces please
and as personal opinion I really prefer 
```php
sprintf('%s/%s', $request->getUri(), $this->cacheProfile->cacheNameSuffix($request))
```